### PR TITLE
Fancy search results view

### PR DESCRIPTION
**Pull request by ima007**
Only tested this in the iPhone 6 simulator, since it's the superior phone. :dancer: 

The business name will wrap a maximum of two lines before being truncated with ellipses.

Screenshot:
Imgur

### CODE REVIEW

**Comment by ima007** [comment]
Ok...i was a little curious, and I looked on the iPhone 6 plus and 5s simulators... :+1:

**Comment by derrickshowers** [comment]
This is awesome! I'll take a look in a bit.

**Comment by derrickshowers** [comment]
I like the addition of the 'Start typing to search' and 'No results' messages. I did notice, though, that if you touch the cell the app crashes with a 'Cannot index empty buffer' error. Looks like maybe the `indexPath` object in the `didSelectRowAtIndexPath` method is empty in that scenario?

**Comment by derrickshowers** [comment]
Oh, and I checked all simulators. Even works for those crazy 4s peeps. :+1: :+1:

File: RecommendIt/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -419,10 +518,14 @@
...
                             <constraint firstAttribute="trailing" secondItem="Y93-vH-F5J" secondAttribute="trailing" id="KkA-EE-7Fp"/>
+                            <constraint firstItem="VtQ-cB-0Zy" firstAttribute="top" secondItem="ijY-03-QUR" secondAttribute="bottom" id="Nob-69-7LD"/>
+                            <constraint firstItem="ijY-03-QUR" firstAttribute="top" secondItem="my8-Qn-cdl" secondAttribute="top" id="Ono-px-k1l"/>
                             <constraint firstItem="tOP-FS-2lU" firstAttribute="top" secondItem="AZo-9m-KHH" secondAttribute="bottom" id="TsI-OI-0kB"/>
                             <constraint firstItem="ijY-03-QUR" firstAttribute="leading" secondItem="my8-Qn-cdl" secondAttribute="leading" id="VXm-Kk-SSY"/>
                             <constraint firstItem="Y93-vH-F5J" firstAttribute="leading" secondItem="my8-Qn-cdl" secondAttribute="leading" id="aY9-z3-xRE"/>
+                            <constraint firstItem="ijY-03-QUR" firstAttribute="leading" secondItem="my8-Qn-cdl" secondAttribute="leading" id="idW-HN-eCJ"/>
```
Comment (derrickshowers) [inline RecommendIt/Base.lproj/Main.storyboard]: I think there might be some extra constraints. I get a warning in Xcode:
```
Constraint referencing items turned off in current configuration. Turn off this constraint in the current configuration
```
Followed [these instructions](http://stackoverflow.com/questions/26547399/xcode-storyboard-warning-constraint-referencing-items-turned-off-in-current-con) to find the culprits, and removed them. Seemed to work. (I can fix it after the merge if you prefer.)

File: RecommendIt/RecommendIt-Bridging-Header.h [hunk 2]
```diff
@@ -2,5 +2,6 @@
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
 
+#import "UIImageView+AFNetworking.h"
```
Comment (derrickshowers) [inline RecommendIt/RecommendIt-Bridging-Header.h]: Does anything new need installed via CocoaPods? Or is this something I already had, but not imported?

Comment (ima007) [inline RecommendIt/RecommendIt-Bridging-Header.h]: Nope, no extra pods necessar. AFNetworking is already included in the project for the oauth stuff, this just imports the UIImageView specific helpers.

**Comment by ima007** [comment]
Ok, I think I addressed both the constraint warning thing and the bug involving tapping on the "No results" cell. Let me know if it works for you!

File: RecommendIt/Base.lproj/Main.storyboard (hunk 1 above)
Comment (ima007) [inline RecommendIt/Base.lproj/Main.storyboard]: should be fixed now!

**Comment by derrickshowers** [comment]
Merging and will submit to Apple as 1.2 soon! :+1:

**Pull request merged**